src/abe_policy/policy.rs

```diff
@@ -164,7 +169,10 @@ impl Policy {
         if let Some(dim) = self.dimensions.get(&attr.dimension) {
             dim.attributes
                 .get(&attr.name)
-                .ok_or(Error::AttributeNotFound(attr.name.to_string()))
+                .ok_or(Error::AttributeNotFound(format!(
+                    "{}::{}",
+                    attr.dimension, attr.name
+                )))
```

Comment:
Doesn't Attribute implement Display ?

```suggestion
                .ok_or(Error::AttributeNotFound(attr.to_string()))
```